Magento/azuredeploy.json

```diff
@@ -0,0 +1,514 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "defaultValue": "[resourceGroup().location]",
+            "type": "string"
+        },
+        "mysqlAdministratorLogin": {
+            "defaultValue": "magento",
```

Comment:
can we generate login user name as a variable rather than standalone value.  for security if user does not change this , there can be malicious attempt to access it. something like this

"mysqlAdministratorLogin":[concat('magento',uniquestring(resourceGroup().id))]

Reply:
Changed it to a parameter w/ non-default value